src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NodeService.kt

```diff
@@ -172,6 +172,10 @@ class NodeService(
           .stream()
           .mapToInt { el -> el.transactionAmount.value }
           .sum())
+    logger.info(
+      "Building close payment for cancellation with effective clientId: [{}], activation client is: [{}]",
```

Comment:
can be useful log `transactionId`

Reply:
Solved by https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/319/commits/3ee8c94f06b1fe5b045b5a66712d8a6f3e77fc65